### PR TITLE
Added in Pixel 2XL configuration.

### PR DESCRIPTION
**Pull request by bootlessxfly**
Points to my personal repo for now.

Signed-off-by: Chris White <bootlessxfly@gmail.com>

This is only for the Pixel 2 XL. I can go update my pixel 2 trees and kernel defconfig if you guys want that. I've built for the pixel 2, but have not device to test it. 

Also, this includes my personal device and kernel repos. I am not sure the correct process on adding this into the maruos repo. Let me know if there is antrthing needed on my end.

### CODE REVIEW

**Comment by utzcoz** [comment]
@bootlessxfly Thanks for your work for Pixel 2. It's not recommended to use personal GitHub repo for MaruOS. If you ensure you have finished the porting of Pixel 2, you can contact @pdsouza to grant your permission to create Pixel 2 repositories under MaruOS org account and push your source code Pixel related repositories.

**Comment by bootlessxfly** [comment]
> @bootlessxfly Thanks for your work for Pixel 2. It's not recommended to use personal GitHub repo for MaruOS. If you ensure you have finished the porting of Pixel 2, you can contact @pdsouza to grant your permission to create Pixel 2 repositories under MaruOS org account and push your source code Pixel related repositories.

@utzcoz Ill reach out to @pdsouza about getting this set up. Ill update this PR when that gets sorted out

**Comment by pdsouza** [comment]
Just invited you to the maruos org @bootlessxfly! Once you are a member you can create or fork the Pixel 2 repos directly on our org, much like @utzcoz has done so for Pixel 3a. 

Very exciting to see support landing for the Pixel 2XL and Pixel 2 soon!

**Comment by pintaf** [comment]
Hello @bootlessxfly if you created the device repo under maruos org, maybe you can update your to reflect these changes ?

File: default.xml [hunk 1]
```diff
@@ -50,8 +54,15 @@
   <project path="kernel/google/marlin" name="android_kernel_google_marlin" remote="maruos" />
   <!-- END Sailfish/Marlin - Google Pixel / Google Pixel XL -->
 
+    <!-- START Wahoo/Taimen - Google Pixel 2XL -->
```
Comment (utzcoz) [inline default.xml]: Align your indents

File: default.xml [hunk 2]
```diff
@@ -31,7 +35,7 @@
   <project path="vendor/maruos/blueprints" name="blueprints" remote="maruos" revision="master" />
   <project path="vendor/maruos/installer" name="installer" remote="maruos" revision="master" />
   <project path="vendor/maruos/packages/apps/MaruSettings" name="vendor_maruos_packages_apps_MaruSettings" remote="maruos" />
-  
+
```
Comment (utzcoz) [inline default.xml]: If you don't add new line or delete old line, it's not recommended to modify lines like this one.

File: default.xml [hunk 3]
```diff
@@ -9,6 +9,10 @@
            fetch="https://github.com/maruos"
            revision="maru-0.7" />
 
+  <remote  name="bootlessxfly"
+           fetch="https://github.com/bootlessxfly"
```
Comment (utzcoz) [inline default.xml]: Please push your device repositories to maruos org like Pixel3a.

**Review by utzcoz — CHANGES_REQUESTED** [review]

**Comment by utzcoz** [comment]
@bootlessxfly If you need, I can help you to setup repositories in maruos org. Feel free to ping me if you need any help.

File: default.xml (hunk 3 above)
Comment (bootlessxfly) [reply default.xml]: Hi! I did not see that this PR was dangling for all this time. Sorry about that!

The device trees where pushed to maruos project almost 2 years ago now, but I guess for some unknown reason I never changed that here. I just pushed a update that points to the maruos tree over my private fork. I think this is all that is needed?

File: default.xml [hunk 4]
```diff
@@ -50,6 +50,13 @@
   <project path="kernel/google/marlin" name="android_kernel_google_marlin" remote="maruos" />
   <!-- END Sailfish/Marlin - Google Pixel / Google Pixel XL -->
 
+  <!-- START Wahoo/Taimen - Google Pixel 2XL -->
+  <project path="device/google/taimen" name="android_device_google_taimen" remote="maruos" />
+  <project path="device/google/wahoo" name="android_device_google_wahoo" remote="maruos" />
+  <project path="kernel/google/wahoo" name="android_kernel_google_wahoo" remote="maruos" />
+  <!-- END Wahoo/Taimen - Google Pixel 2XL -->
+
+
```
Comment (utzcoz) [inline default.xml]: @bootlessxfly nit: What about removing this empty line? It's great to merge after this change.

Comment (bootlessxfly) [reply default.xml]: done!

**Comment by pdsouza** [comment]
Thanks @bootlessxfly for the quick fixes! Looking forward to more of your contributions!

**Comment by utzcoz** [comment]
@bootlessxfly merged. Please test the latest 0.7 branch to check whether it works as you expected. It's great news we have Pixel 2L source code supporting now.

**Pull request merged**